tests/wicked/before_test.pm

```diff
@@ -81,6 +81,9 @@ EOT
         if (is_sle('<12-sp1')) {
             file_content_replace('/etc/dhcpd.conf', '^\s*ddns-update-style' => '# ddns-update-style', '^\s*dhcp-cache-threshold' => '# dhcp-cache-threshold');
         }
+        if (my $mtu = get_var('MM_MTU')) {
+            file_content_replace('/etc/dhcpd.conf', 'interface-mtu 1380' => "interface-mtu $mtu");
```

Comment:
Maybe better use some token that clearly is replaced, and always replace that one.
e.g. `%MTU%`

Reply:
I wanted to avoid having to replace anything in case the MTU isn't set anyways.